e2e/testcafe-devextreme/tests/scheduler/common/layout/horizontalScroll/horizontalScroll.ts

```diff
@@ -0,0 +1,169 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
```

Comment:
I suggest to move this test to [this file](https://github.com/DevExpress/DevExtreme/blob/45525faa3bd834615ab7406c114121374fce698e/e2e/testcafe-devextreme/tests/scheduler/common/header/header.ts#L46), because we add `padding-right` for scheduler's header